### ISSUE TITLE
Support for exporting to google bigquery (to_gbq)

### ISSUE THREAD

**Issue by andhuang-CLGX** [geopandas/geopandas#2142]
Not sure if it's more appropriate here:
https://github.com/googleapis/python-bigquery-pandas/issues/400

But basically:
https://geopandas.org/docs/reference/api/geopandas.GeoDataFrame.html lists to_gbq, but I think it just inherits pandas-gbq; is there any plans to support a geometry column to bigquery

**Comment by martinfleis** [comment]
Thanks for the suggestion! I think that is on us to override `to_gbq` to ensure that geometry gets converted properly. There are no specific plans on that but contributions are welcome :).

**Comment by martinfleis** [comment]
Dropping this here as a reference https://cloud.google.com/bigquery/docs/geospatial-data

It seems that we need to ensure lat/lon in WGS84 and then we can go via WKT/WKB as we do with PostGIS.

**Comment by martinfleis** [comment]
Looking into the pandas implementation, it actually is probably not something to be implemented here. pandas uses `pandas_gbq` to take care of complete IO. If the geometry support is implemented there, we don't have to do anything.

**Comment by jorisvandenbossche** [comment]
Yes, I think it would be nice if this could be supported by https://github.com/googleapis/python-bigquery-pandas (they could check if they receive a GeoDataFrame without having a hard dependency on geopandas).

**Comment by andhuang-CLGX** [comment]
I tried uploading thru gbq with no changes to my geopandas dataframe (e.g. the geometry column is automatically converted to WKT) and it just works as long as the table_schema explicitly specifies the geometry column as "GEOGRAPHY" type.
